### ISSUE TITLE
Is whalesong whatsapp can detect when someone new want to send message to you?

### ISSUE THREAD

**Issue by DanielUnn** [alfred82santa/whalesong#23]
Is whalesong whatsapp can detect when someone new want to send message to you, whalesong able to send message to that person right away?

Is there any indication for this? Thanks. @alfred82santa

**Comment by alfred82santa** [comment]
I don't understand, sorry. Do you mean when someone is typing message?

**Comment by DanielUnn** [comment]
Like when you added someone who is your new friend, you want to send a whatsapp message to him.
You will search his name on whatsapp and click on it in order to message him.

So I wondering, when you click the chat box of him, I want whalesong auto send him a message such as 'Welcome to ABC group. Our society is doing bla bla bla.' before any of our conversation starts. Can we do that?

**Comment by alfred82santa** [comment]
Well, it is not a whalesong issue. It is how whatsapp works. User chats (not groups) are created on receiver with first message, not before.

**Comment by DanielUnn** [comment]
Yeah it is not whalesong issue. So we not able to do that as this is how whatsapp works?

**Comment by alfred82santa** [comment]
exactly
